README.md

```diff
@@ -397,6 +397,14 @@ Type: _string_
 
 Default: `''`
 
+#### `custom_runs_on`
+
+GitHub Actions runner type for custom jobs.
```

Comment:
Why do we need two separate `runs_on` entries?

Reply:
Presumably that's only a problem for a handful that we want to be run as singletons, eg `Event types`/`Versioned Source`/`Protect Branch` but for others we may want to do multi-arch like `Test npm` in order to test on multiple OS versions. I'm thinking it may make more sense to have the distinction of `runs_on` vs `tests_runs_on`?

Reply:
@Page- do you want me to rename `custom_runs_on` to `tests_runs_on`?